enclave/src/lib.rs

```diff
@@ -1046,12 +1101,13 @@ fn execute_ocex_release_extrinsic(
     let xt_block = [OCEX_MODULE, OCEX_RELEASE];
     let genesis_hash = validator.genesis_hash(validator.num_relays).unwrap();
     let call: OpaqueCall = OpaqueCall((xt_block, token, amount, acc).encode());
+    let nonce = lock_and_get_finalized_nonce()?;
```

Comment:
Make sure the lock is implemented inside this function and lock the nonce after getting the signer key

Reply:
done